doc/topics/releases/neon.rst

```diff
@@ -3,3 +3,9 @@
 ==================================
 Salt Release Notes - Codename Neon
 ==================================
+
+Module Changes
+==============
+
+- The :py:func:`freezer.restore <salt.modules.freezer.restore>` function provides a
+  new parameter, `clean`, that will remove the frozen data after a successful recovery.
```

Comment:
We can probably remove this comment here, since the `freezer` module was never released, right?

Reply:
You are right. Done